profiler/src/ProfilerEngine/Datadog.Profiler.Native/Configuration.cpp

```diff
@@ -93,6 +93,7 @@ Configuration::Configuration()
     _isEtwEnabled = GetEnvironmentValue(EnvironmentVariables::EtwEnabled, false);
     _deploymentMode = GetEnvironmentValue(EnvironmentVariables::SsiDeployed, DeploymentMode::Manual);
     _isEtwLoggingEnabled = GetEnvironmentValue(EnvironmentVariables::EtwLoggingEnabled, false);
+    _etwEndpoint = GetEnvironmentValue(EnvironmentVariables::EtwEndpoint, DefaultEmptyString);
```

Comment:
is the `_etwEndpoint` only for the replay scenario ? if yes maybe make it clear :)